tests/array_interoperability_test.py

```diff
@@ -215,7 +212,7 @@ def testNumpyToJax(self, shape, dtype):
      for shape in all_shapes
      for dtype in torch_dtypes))
   @unittest.skipIf(numpy_version < (1, 22, 0), "Requires numpy 1.22 or newer")
-  @jtu.skip_on_devices("gpu")
+  @jtu.skip_on_devices("gpu") # Numpy doesn't support DLpacks from GPU vendors
```

Comment:
A better way to phrase this would be:
"NumPy only accepts cpu DLPacks".

Reply:
Done!